lib/msf/core/payload/jsp.rb

```diff
@@ -21,11 +21,12 @@ def initialize(info = {})
   # @return [String] jsp code that executes bind TCP payload
   def jsp_bind_tcp
     # Modified from: http://www.security.org.sg/code/jspreverse.html
+    generator = Rex::RandomIdentifier::Generator.new({ language: :jsp })
```

Comment:
https://github.com/rapid7/rex-random_identifier/pull/6

The PR above needs to land before this code works.